String_Lab.js

```diff
@@ -0,0 +1,184 @@
+
+// 1. console.log the first character of a string.
+let string = "Lesson";
+console.log("Printing the first character of the word ", string);
+console.log(string[0]);
+console.log("---------------------------------- \n");
+
+
+// 2. console.log the length of a string.
+console.log(`The lenght of ${string} is ${string.length}`)
+console.log("---------------------------------- \n");
+
+
+// 3. console.log the last character of any string.
+console.log(`The last character of the word ${string} is ${string[string.length - 1]}`);
+console.log("---------------------------------- \n");
+
+
+// 4. Create a code block that takes a single string variable, and returns
+// a copy of the string with the last letter capitalized. For example:
+
+// 'jimmy'
+// => 'jimmY'
+console.log(`Capitalizing the last character of the word ${string} :`);
+console.log("Methode1 : ")
+console.log(string.slice(0, string.length - 1) + string[string.length - 1].toUpperCase());
+console.log("Methode2 : ")
+console.log(`${string}\b${string[string.length - 1].toUpperCase()}`);
+console.log("---------------------------------- \n");
+
+
+// 5 Create a drEvil code block that will take a single number varaible, and log
+// the ' dollars', your code block should add '(pinky)' at the end of
+// the amount if it's 1 million. For example:
+
+// let amount = 10
+// => 10 dollars
+// let amount = 1000000
+// =>  1000000 dollars (pinky)
+let amount = 1000000;
+if (amount === 1000000) {
+console.log(`Amount of money :\n\t ${amount} dollars (pinky) ... Wish I had that in my bank account :)`);
```

Comment:
Lines inside a code block (body) of an `if`, `for` or `while` statements should be indented to the right to indicate that they are inside something else (the body/code block of a statement)